vllm/model_executor/models/deepseek_v2.py

```diff
@@ -553,6 +553,8 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
 
         params_dict = dict(self.named_parameters())
         for name, loaded_weight in weights:
+            if "weight_shape" in name:
```

Comment:
Could you make this more specific? At least should be able to upgrade to `.weight_shape`

Also what is this added from?

Reply:
This was added because compressed-tensors gives us weight_shapes which we never use. In all the other cases, we just load the parameter and then dont use it during the forward pass. Added this as a quick workaround but will add the parameters to be consistent.